licensescout-parent/pom.xml

```diff
@@ -195,7 +196,16 @@
 	</scm>
 
 	<dependencyManagement>
+
 		<dependencies>
+
+			<!-- groupId: commons-codec -->
+			<dependency>
+				<groupId>commons-codec</groupId>
+				<artifactId>commons-codec</artifactId>
+				<version>${commons-codec.version}</version>
```

Comment:
This dependency wasn't here before and is not used explicitly anywhere...is it a transitive dependency? If so, could you instead update the other dependency? If not, then add a comment saying that this is a transitive dependency added for security reasons...

Reply:
Yes, it is a transitive dependency. I added more information on the dependency paths in the description and linked a general description of the security issue.
Conclusion is: "Newer versions of doxia and httpclient still depend on the same version of commons-codec, so locking the version of the transitive dependency is currently the only possible solution."

Reply:
Could you add the reference and the comment to the pom? That was what I meant on the comment, as like that we don't risk to clean "unused" dependencies that are security relevant...

Reply:
OK, understood. I added a remark in the POM.